mne/epochs.py

```diff
@@ -402,6 +409,8 @@ def decimate(self, decim, copy=False):
         else:
             epochs._decim_slice = decim_slice
             epochs.times = epochs._raw_times[epochs._decim_slice]
+        epochs.tmin = epochs.times[0]
+        epochs.tmax = epochs.times[-1]
```

Comment:
Since you opened this PR, `tmin` and `tmax` have become properties:

https://github.com/mne-tools/mne-python/blob/master/mne/epochs.py#L1292

So you shouldn't need to set them as long as `.times` and `._raw_times` are set properly.